src/component/calculator.js

```diff
@@ -1,8 +1,8 @@
-import React, { Component } from 'react';
+import React from 'react';
 /* eslint-disable */
 
```

Comment:
**[OPTIONAL BUT HIGHLY RECOMMENDED]**

- [ ] Apologies on behalf of the previous reviewer but it would be great if you removed this eslint disable line because eslint prevents you from introducing those errors in your app which might break your app completely 😨